package.json

```diff
@@ -74,6 +75,7 @@
   "devDependencies": {
     "@keystonehq/bc-ur-registry-eth": "^0.9.0",
     "@lavamoat/allow-scripts": "^2.0.2",
+    "@ledgerhq/hw-transport": "^6.24.1",
```

Comment:
This is used in `src/keyring/KeyringController.ts`, so it should be a regular dependency rather than a dev dependency

Reply:
[b15d7fc](https://github.com/MetaMask/controllers/pull/862/commits/b15d7fc622523e72e99bf200829345120b786561)